### PR TITLE
Removing 3 gwei floor on 1559 gas price estimate

### PR DESCRIPTION
**Pull request by aroralanuk**
## PR Checklist

-   [ ] Added Tests
-   [ ] Added Documentation
-   [ ] Updated the changelog

### CODE REVIEW

File: ethers-core/src/utils/mod.rs [hunk 1]
```diff
@@ -486,8 +481,8 @@ fn estimate_priority_fee(rewards: Vec<Vec<U256>>) -> U256 {
 
     // If we encountered a big change in fees at a certain position, then consider only
     // the values >= it.
-    let values = if *max_change >= EIP1559_FEE_ESTIMATION_THRESHOLD_MAX_CHANGE.into() &&
-        (max_change_index >= (rewards.len() / 2))
+    let values = if *max_change >= EIP1559_FEE_ESTIMATION_THRESHOLD_MAX_CHANGE.into()
+        && (max_change_index >= (rewards.len() / 2))
```
Comment (yorhodes) [inline ethers-core/src/utils/mod.rs]: nit: revert

File: ethers-core/src/utils/mod.rs [hunk 2]
```diff
@@ -413,7 +413,7 @@ pub fn to_checksum(addr: &Address, chain_id: Option<u8>) -> String {
 pub fn format_bytes32_string(text: &str) -> Result<[u8; 32], ConversionError> {
     let str_bytes: &[u8] = text.as_bytes();
     if str_bytes.len() > 32 {
-        return Err(ConversionError::TextTooLong)
+        return Err(ConversionError::TextTooLong);
```
Comment (yorhodes) [inline ethers-core/src/utils/mod.rs]: nit: revert

File: ethers-core/src/utils/mod.rs [hunk 3]
```diff
@@ -434,15 +434,10 @@ pub fn parse_bytes32_string(bytes: &[u8; 32]) -> Result<&str, ConversionError> {
...
-            U256::from(EIP1559_FEE_ESTIMATION_DEFAULT_PRIORITY_FEE)
-        } else {
-            std::cmp::max(
-                estimate_priority_fee(rewards),
-                U256::from(EIP1559_FEE_ESTIMATION_DEFAULT_PRIORITY_FEE),
-            )
-        };
+    let max_priority_fee_per_gas = std::cmp::max(
+        estimate_priority_fee(rewards),
+        U256::from(EIP1559_FEE_ESTIMATION_DEFAULT_PRIORITY_FEE),
```
Comment (yorhodes) [inline ethers-core/src/utils/mod.rs]: lower the default if this is a max?

File: ethers-core/src/utils/mod.rs [hunk 4]
```diff
@@ -434,15 +434,10 @@ pub fn parse_bytes32_string(bytes: &[u8; 32]) -> Result<&str, ConversionError> {
 
 /// The default EIP-1559 fee estimator which is based on the work by [MyCrypto](https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/services/ApiService/Gas/eip1559.ts)
 pub fn eip1559_default_estimator(base_fee_per_gas: U256, rewards: Vec<Vec<U256>>) -> (U256, U256) {
-    let max_priority_fee_per_gas =
-        if base_fee_per_gas < U256::from(EIP1559_FEE_ESTIMATION_PRIORITY_FEE_TRIGGER) {
```
Comment (tkporter) [inline ethers-core/src/utils/mod.rs]: having a hard time understanding why this was fine to remove

**Comment by tkporter** [comment]
Reverted the formatting issues and added back the `if base_fee_per_gas < U256::from(EIP1559_FEE_ESTIMATION_PRIORITY_FEE_TRIGGER) {` logic. Lmk if this was removed for a strong reason - just need to merge this because I'll be pulling some upstream changes into our for

**Pull request merged**